ci-operator/config/openshift-kni/baremetal-deploy/openshift-kni-baremetal-deploy-master.yaml

```diff
@@ -0,0 +1,25 @@
+base_images:
+  base:
+    name: "4.3"
+    namespace: ocp
+    tag: base
+build_root:
```

Comment:
this build_root is fine for now.

fyi for later thought. 

If we find that we want to create a custom build_root that has our own specialized tooling in it for some reason (like having ginkgo pre-loaded for example), we could make a custom build_root from a Dockerfile in our repo.

```
build_root:
  project_image:
    dockerfile_path: openshift-ci/Dockerfile.tools
```

^ that would create the build_root using a Dockerfile in the openshift-ci dir of baremetal-deploy which would then be used for all the other steps, like the sctp e2e test for example.

Reply:
Thanks!